### PR TITLE
Minor follow-up to #984

### PR DESCRIPTION
**Pull request by julien-c**

### CODE REVIEW

File: docs/hub/models-libraries.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-| [🤗 Diffusers](https://github.com/huggingface/diffusers)                    | A modular toolbox for inference and training of diffusion models                     | ❌ | ❌ | ✅ | ✅ |
 | [Adapter Transformers](https://github.com/Adapter-Hub/adapter-transformers) | Extends 🤗Transformers with Adapters.                                                | ❌ | ❌ | ✅ | ✅ |
 | [AllenNLP](https://github.com/allenai/allennlp)                             | An open-source NLP research library, built on PyTorch.                               | ✅ | ✅ | ✅ | ❌ |
 | [Asteroid](https://github.com/asteroid-team/asteroid)                       | Pytorch-based audio source separation toolkit                                        | ✅ | ✅ | ✅ | ❌ |
 | [BERTopic](https://github.com/MaartenGr/BERTopic)                           | BERTopic is a topic modeling library for text and images                             | ✅ | ✅ | ✅ | ✅ | 
+| [Diffusers](https://github.com/huggingface/diffusers)                       | A modular toolbox for inference and training of diffusion models                     | ❌ | ❌ | ✅ | ✅ |
```
Comment (osanseviero) [inline docs/hub/models-libraries.md:14]: Let's change transformers as well please

**Review by osanseviero — COMMENTED** [review]
Thanks!

File: docs/hub/models-libraries.md (hunk 1 above)
Comment (julien-c) [reply docs/hub/models-libraries.md:14]: keeping it first still, or not?

**Review by sayakpaul — APPROVED** [review]
Thank you!

File: docs/hub/models-libraries.md (hunk 1 above)
Comment (osanseviero) [reply docs/hub/models-libraries.md:14]: I would say no to keep alphabetical order

Comment (pcuenca) [reply docs/hub/models-libraries.md:14]: I'd maybe keep it first in the table.

Comment (julien-c) [reply docs/hub/models-libraries.md]: no super strong opinion, i've sorted it alphabetically too for now 05ba4b16a2d5edbe36cd22277f9ba9c77cc22da3

File: docs/hub/models-libraries.md [hunk 2]
```diff
@@ -23,7 +22,7 @@ The table below summarizes the supported libraries and their level of integratio
 | [PaddleNLP](https://github.com/PaddlePaddle/PaddleNLP)                      | Easy-to-use and powerful NLP library built on PaddlePaddle                           | ✅ | ✅ | ✅ | ✅ |
 | [Pyannote](https://github.com/pyannote/pyannote-audio)                      | Neural building blocks for speaker diarization.                                      | ❌ | ❌ | ✅ | ❌ |
 | [PyCTCDecode](https://github.com/kensho-technologies/pyctcdecode)           | Language model supported CTC decoding for speech recognition                         | ❌ | ❌ | ✅ | ❌ |
-| [Pythae](https://github.com/clementchadebec/benchmark_VAE)                  | Unifyed framework for Generative Autoencoders in Python | ❌ | ❌ | ✅ | ✅ |
+| [Pythae](https://github.com/clementchadebec/benchmark_VAE)                  | Unified framework for Generative Autoencoders in Python | ❌ | ❌ | ✅ | ✅ |
```
Comment (pcuenca) [inline docs/hub/models-libraries.md:25]: 👍

**Review by pcuenca — APPROVED** [review]

**Review by osanseviero — APPROVED** [review]
Thanks! Let's merge it

**Pull request merged**